lib/screens/home/accounts/accounts_bloc.dart

```diff
@@ -17,7 +17,7 @@ class AccountsBloc implements Disposable {
 
   final _subscriptions = CompositeSubscription();
   final _accountService = get<AccountService>();
-  final _assetService = get<AssetService>();
+  final _assetService = get<AssetClient>();
```

Comment:
Oops, missed this local field. Wondering if there might also be any `assetService` fields that a text search could turn up?

Reply:
A few, as it turns out.